Bottom: 9eec67024c3c2c75c707812d1eb8fb3c06ceff73
Top:    97e4f88000b5fc065cccbf8ee26d6e3ee483792c
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-18 14:12:14 -0500

Refresh of TESTING--VirtualMachine

---
diff --git a/roles/docker-exim4/tasks/main.yml b/roles/docker-exim4/tasks/main.yml
index 052ad8c..f286d43 100644
--- a/roles/docker-exim4/tasks/main.yml
+++ b/roles/docker-exim4/tasks/main.yml
@@ -87,6 +87,7 @@
     name: '{{ exim4_image_name }}'
     tag: '{{ docker_image_tag }}'
     path: '{{ docker_projects_dir }}/docker-exim4'
+    force: true
 
 # *****************************************************************************
 # Create the data volumes
diff --git a/roles/docker-mantisbt/tasks/main.yml b/roles/docker-mantisbt/tasks/main.yml
index 3ee9591..0c2f902 100644
--- a/roles/docker-mantisbt/tasks/main.yml
+++ b/roles/docker-mantisbt/tasks/main.yml
@@ -81,6 +81,7 @@
     name: '{{ mantisbt_image_name }}'
     tag: '{{ docker_image_tag }}'
     path: '{{ docker_projects_dir }}/docker-mantisbt'
+    force: true
 
 - name: Checkout db repo
   git:
diff --git a/roles/docker-mediawiki/tasks/main.yml b/roles/docker-mediawiki/tasks/main.yml
index 580c03f..582a1fa 100644
--- a/roles/docker-mediawiki/tasks/main.yml
+++ b/roles/docker-mediawiki/tasks/main.yml
@@ -82,6 +82,7 @@
     name: '{{ wiki_image_name }}'
     tag: '{{ docker_image_tag }}'
     path: '{{ docker_projects_dir }}/docker-mediawiki'
+    force: True
 
 # *****************************************************************************
 # Create the data volumes
diff --git a/roles/docker-openldap/tasks/main.yml b/roles/docker-openldap/tasks/main.yml
index 125fc4f..45c83c2 100644
--- a/roles/docker-openldap/tasks/main.yml
+++ b/roles/docker-openldap/tasks/main.yml
@@ -72,6 +72,7 @@
     name: '{{ openldap_image_name }}'
     tag: '{{ docker_image_tag }}'
     path: '{{ docker_projects_dir }}/docker-openldap'
+    force: true
 
 # *****************************************************************************
 # Create the data volumes
@@ -136,6 +137,7 @@
     name: '{{ phpldapadmin_image_name }}'
     tag: '{{ docker_image_tag }}'
     path: '{{ docker_projects_dir }}/docker-phpldapadmin'
+    force: true
 
 - name: start container (phpldapadmin)
   docker_container:
diff --git a/roles/docker-svn/tasks/main.yml b/roles/docker-svn/tasks/main.yml
index 2fc8540..40661b6 100644
--- a/roles/docker-svn/tasks/main.yml
+++ b/roles/docker-svn/tasks/main.yml
@@ -61,10 +61,10 @@
 
 - name: build image
   docker_image:
-    force: true
     name: '{{ svn_image_name }}'
     tag: '{{ docker_image_tag }}'
     path: '{{ docker_projects_dir }}/docker-svn'
+    force: true
 
 # *****************************************************************************
 # Create the data volumes
